source/server/admin/admin.cc

```diff
@@ -455,15 +461,19 @@ Http::Code AdminImpl::handlerClusters(absl::string_view url,
 }
 
 void AdminImpl::addAllConfigToDump(envoy::admin::v3::ConfigDump& dump,
-                                   const absl::optional<std::string>& mask,
-                                   bool include_eds) const {
+                                   const absl::optional<std::string>& mask, bool include_eds,
+                                   bool include_listener_unready_targets) const {
```

Comment:
Since the API is no longer listener specific, should we make this one generic as well? (through out this file)

Reply:
Emm, but actually the current ```dumoUnreadyTargetsConfigs``` only dumps unready targets of (warming) listeners

Reply:
How about in long term? We're likely having more unready targets when we address ECDS / On-demand DS / asynchronous Filter initialization?

Reply:
I am not sure how do we specify whose unready targets do we want. And how do we track their init manager in a general approach.

Reply:
Updated. Added a general method:  ```dumpUnreadyTargetsConfig(optional::string mask)``` to use mask to decide what to dump.